mongo/tests/test_integration.py

```diff
@@ -651,5 +652,12 @@ def test_mongod_tls_fail(check, dd_run_check, aggregator):
         'tls_ca_file': '{}/ca.pem'.format(TLS_CERTS_FOLDER),
     }
     mongo_check = check(instance)
-    dd_run_check(mongo_check)
+    with pytest.raises(
+        Exception,
+        match=(
+            "pymongo.errors.ConfigurationError:"
+            r" Private key doesn't match certificate: \[SSL\] PEM lib \(_ssl.c:4065\)"
```

Comment:
I'm not sure what's the right amount of exception text that we should be matching, but I think we'd better trim this a little bit here to at least not include references to code files that we don't own.

```suggestion
            r" Private key doesn't match certificate"
```